Automattic/pocket-casts-android
# Transcripts -  Integrate (temporary) UI container to show transcript in the Player

## Description
This displays an episode transcript in the player container's "temporary" `Transcripts` tab.

## Testing Instructions

1. Launch the app
2. Play an episode from "Cautionary Tales" (https://pcast.pocketcasts.net/8zpmjiru)
3. Open player tab
4. ✅ Notice there's a new `Transcripts` tab
5. Tap on the `Transcripts` tab
6. ✅ Notice that the episode's transcript is displayed on the tab (ignore all styling)
7. Play an episode that does not have a transcript 
8. ✅ Notice `Transcripts` tab is not shown for the episode

## Screenshots or Screencast 

<image width=250 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/abaf2dc9-6b8a-425a-b50d-1898d790f9c3"/>


## Checklist
-  If this is a user-facing change, I have added an entry in CHANGELOG.md - N/A
- Any jetpack compose components I added or changed are covered by compose previews - Not added as views are temporary
- I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics. - TBD 
 
#### I have tested any UI changes...  


(N/A as views are temporary)
- with different themes
- with a landscape orientation
- with the device set to have a large display and font size
- for accessibility with TalkBack
